opm/flowdiagnostics/utility/AssembledConnections.cpp

```diff
@@ -256,9 +256,7 @@ CSR::accumulateRowEntries(const int               numRows,
     }
 
     // Note index range: 1..numRows inclusive.
-    for (decltype(this->ia_.size())
-             i = 1, numRows = this->ia_.size() - 1;
-         i <= numRows; ++i)
+    for (int i = 1; i <= numRows; ++i)
```

Comment:
I think we'll need to have a deeper discussion about this, we obviously disagree quite a bit. Let's leave it for now and pick it up when we can discuss it in person.